docs/manual/docs/cmd/spo/contenttype/contenttype-field-remove.md

```diff
@@ -0,0 +1,51 @@
+# spo contenttype field remove
```

Comment:
This file needs to be referenced from `docs/manual/mkdocs.yml` or it won't be included in the generated docs.